### PR TITLE
Update Getting shopping cart ID procedure

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [X] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

PR created to update  procedure "Getting shopping cart ID" (EDU-9024).

### CODE REVIEW

File: docs/guides/Checkout/shopping-cart-section-api-quick-start-guides/get-cart-information-by-id.md [hunk 1]
```diff
@@ -53,14 +85,8 @@ The following error may appear as a message in the response body.
...
-{
-  "codes": [
-    {
-      "code": "<body>\n\t<h1>404 Not Found</h1>\n\t<p>The requested URL was not found on this server.</p>\n</body>",
-      "language": "json"
-    }
-  ]
-}
-[/block]
+- `Message error example: "The requested URL was not found on the server"`. Check that the URL data is correct.
```
Comment (karenkrieger) [inline docs/guides/Checkout/shopping-cart-section-api-quick-start-guides/get-cart-information-by-id.md:88]:
```suggestion
- `Message error example: "The requested URL was not found on the server"`: check that the URL data is correct.
```

File: docs/guides/Checkout/shopping-cart-section-api-quick-start-guides/get-cart-information-by-id.md [hunk 2]
```diff
@@ -53,14 +85,8 @@ The following error may appear as a message in the response body.
...
+
+```html
+https://developers.vtex.com/docs/guides/orderform-fields
```
Comment (karenkrieger) [inline docs/guides/Checkout/shopping-cart-section-api-quick-start-guides/get-cart-information-by-id.md:91]: O erro fica com o link da documentação?

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: docs/guides/Checkout/shopping-cart-section-api-quick-start-guides/get-cart-information-by-id.md (hunk 2 above)
Comment (ricardoaerobr) [reply docs/guides/Checkout/shopping-cart-section-api-quick-start-guides/get-cart-information-by-id.md]: Não, copiei errado mesmo. Muito obrigado!

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**